src/Caching/Hybrid/src/Internal/DefaultJsonSerializerFactory.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Caching.Hybrid.Internal;
+internal sealed class DefaultJsonSerializerFactory : IHybridCacheSerializerFactory
+{
+    public bool TryCreateSerializer<T>([NotNullWhen(true)] out IHybridCacheSerializer<T>? serializer)
+    {
+        // no restriction
+        serializer = new DefaultJsonSerializer<T>();
```

Comment:
nit: looks like this is a stateless type so could be cached

Reply:
@BrennanConroy  this is the stupidest implementation that is only intended to prove that the configuration API works

Reply:
😆 ok, wasn't sure what was real and what was temporary